artemis/config.py

```diff
@@ -611,6 +611,29 @@ class DomainExpirationScanner:
                 int, "The scanner warns if the domain's expiration date falls within this time frame from now."
             ] = get_config("DOMAIN_EXPIRATION_TIMEFRAME_DAYS", default=30, cast=int)
 
+        class SqlInjectionDetector:
+            SQL_INJECTION_TIME_THRESHOLD: Annotated[
+                int,
+                "Seconds to method sleep() or pg_sleep()",
+            ] = get_config("THRESHOLD", default=2, cast=int)
+
+            SQL_INJECTION_SLEEP_PAYLOADS: Annotated[
+                List[str],
+                "List of payloads with sleep() and pg_sleep() functions.",
+            ] = get_config(
+                "SLEEP_PAYLOAD",
+                default=[
+                    f"'||sleep({SQL_INJECTION_TIME_THRESHOLD})||'",
+                    f"'||pg_sleep({SQL_INJECTION_TIME_THRESHOLD})||'",
+                ],
+                cast=list,
+            )
+
+            SQL_INJECTION_ERROR_PAYLOADS: Annotated[
+                List[str],
+                "Payloads to be substituted for parameter values when creating a URL with a payload.",
+            ] = get_config("PAYLOAD", default=["'", '"'], cast=list)
```

Comment:
`PAYLOAD` -> `SQL_INJECTION_ERROR_PAYLOADS`, again this doesn't have to be config